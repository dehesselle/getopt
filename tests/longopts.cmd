#!/bin/sh
"$1" -o a -l long1,secondlong:,thirdlong::,abbriviation: -- --long1 EXTRA1 --thirdlong=ARGUMENT --secondlong=ARG --abbr ABBRARG -a
