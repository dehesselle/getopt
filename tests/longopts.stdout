 --long1 --thirdlong 'ARGUMENT' --secondlong 'ARG' --abbriviation 'ABBRARG' -a -- 'EXTRA1'
